src/Pages/yechanhong/Login/Login.js

```diff
@@ -1,10 +1,152 @@
 import React, { Component } from "react";
-export class LoginHong extends Component {
+import "./Login.scss";
+import { Link } from "react-router-dom";
+
+class LoginHong extends Component {
+  constructor() {
+    super();
+    this.state = {
+      loginVaildation: false,
+      loginValue: "",
+      passwordValue: "",
+    };
+  }
+
+  handleIdValue = (e) => {
+    this.setState({
+      loginValue: e.target.value,
+    });
```

Comment:
비구조화를 이용하면 더 간결해집니다! 앞으로 다 적용해주세요!
const {value} = e.target;
```suggestion
    this.setState({
      loginValue: value,
    });
```

Reply:
비구조화를 사용헤서 refactoring 했습니다!